docs/howto/upgrade-version.txt

```diff
@@ -42,6 +42,31 @@ released or if some of your dependencies are not well-maintained, some of your
 dependencies may not yet support the new Django version. In these cases you may
 have to wait until new versions of your dependencies are released.
 
+Resolving deprecation warnings
+==============================
+
+Before starting an upgrade it's important to resolve any existing deprecation
+warnings that are raised by your project. Working through these warnings prior
+to upgrading ensures that you're fully informed about areas of the code that
+need altering.
+
+In Python 2.7+, deprecation warnings are silenced by default. It is important
```

Comment:
I think we can drop "2.7+" at this point since 2.6 is mostly gone from the Django ecosystem.
It is -> It's